pkg/util/oom/oom_linux.go

```diff
@@ -33,12 +33,10 @@ import (
 )
 
 func NewOOMAdjuster() *OOMAdjuster {
-	oomAdjuster := &OOMAdjuster{
+	return &OOMAdjuster{
 		pidLister:        getPids,
-		ApplyOOMScoreAdj: applyOOMScoreAdj,
+		applyOOMScoreAdj: applyOOMScoreAdj,
```

Comment:
Can you share more about why these are becoming private?

Reply:
In the previous `struct OOMAdjuster`, it has 2 exported methods: `ApplyOOMScoreAdj` and `ApplyOOMScoreAdjContainer`. So I define the `Interface` which wraps these two methods and now `struct OOMAdjuster` implements this interface. `applyOOMScoreAdj` is a data member of `struct OOMAdjuster` and is called in member function `ApplyOOMScoreAdj`,  because we cannot  have a  data member and member function both named as `ApplyOOMScoreAdj`